src/Windows/local_cache.cpp

```diff
@@ -224,12 +224,24 @@ wil::unique_hfile OpenHandle(LPCWSTR lpFileName, DWORD dwDesiredAccess, DWORD dw
     
     wil::unique_hfile file;
     int i = 0;
+    bool retry;
```

Comment:
[nit]: lines 226-244 could be reduced with something like this:
```
bool retry = true;
for (int i=0; (i<MAX_RETRY) && retry; ++i) {
    if (i > 0) Sleep(SLEEP_RETRY_MS);
    file.reset(CreateFile(lpFileName, dwDesiredAccess, dwShareMode, lpSecurityAttributes,
        dwCreationDisposition, dwFlagsAndAttributes, hTemplateFile));
    retry = !file && (GetLastError() == ERROR_SHARING_VIOLATION);
}
```

Reply:
My personal Nit preference is to not have if blocks without { } to ensure no user error is made when editing/debugging code.

Reply:
sure thing, this is just a suggestion, and it is about using the Boolean variable `retry` as it is (i.e. instead of  `if x  is true then foo is true` -> `foo is x` ) + for-loop because there is a check for the upper boundary (`MAX_RETRY`)